rotkehlchen/globaldb/cache.py

```diff
@@ -7,6 +7,12 @@
 from rotkehlchen.utils.misc import ts_now
 
 
+UNIQUE_CACHE = 'unique_cache'
+GENERAL_CACHE = 'general_cache'
+UNIQUE_CACHE_KEYS = ['TEST_KEY']
+# should be ["CURVE_POOL_ADDRESS", "MAKERDAO_VAULT_ILK", "CURVE_GAUGE_ADDRESS"]
```

Comment:
why having test_key here? shouldn't it take the values in the comment? Edit this and adjust where needed